config/features/config.go

```diff
@@ -130,7 +131,11 @@ func configureTestnet(ctx *cli.Context) error {
 		}
 		params.UseRopstenNetworkConfig()
 	} else {
-		log.Warn("Running on Ethereum Consensus Mainnet")
+		if ctx.IsSet(cmd.ChainConfigFileFlag.Name) {
+			log.Warn("Running on Ethereum network with chainconfig")
```

Comment:
Is there a way we can figure out which network it is from the yaml file itself? If not then maybe we can say "Running on custom Ethereum network specified in a chain configuration yaml file"

Reply:
The `chain-config-file` is unmarshalled after this code is called. 
Therefore we cannot get the value at this point and only have the default  `mainnet` that may not match the config value.

A way around this would be to reorganize the code below to load the file before or log after. https://github.com/prysmaticlabs/prysm/blob/fce9e6883db7e5452b1469db714f73ea4b5eccab/beacon-chain/node/node.go#L123-L130

Reply:
That might be a more invasive change that could break stuff. I would recommend going with your approach and saying something like "Running on custom Ethereum network specified in a chain configuration yaml file"

Reply:
done!